src/geometry/euclidean/sphere/basic.lean

```diff
@@ -150,24 +146,49 @@ begin
   simp
 end
 
+end metric_space
+
+section normed_space
```

Comment:
The lemmas in this section have been weakened from `inner_product_space ℝ V` to `normed_space ℝ V`.
This isn't motivated by the linter, but seems like an acceptable generalization.